std_msgs/Header header
actionlib_msgs/GoalID goal_id
---
uint8 SUCCEEDED=0
uint8 FAILED=1
uint8 result_status